code/src/components/ClearButton.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useDispatch } from "react-redux"
+import { tasks } from 'reducers/Tasks'
```

Comment:
I think the convention is to group all import from "outside" together and all local imports one line below